builders/toolkit/rpc.md

```diff
@@ -0,0 +1,102 @@
+---
+title: JSON-RPC API Methods
+description: Learn about the supported JSON-RPC API methods for querying your Tanssi EVM appchain, including standard Ethereum methods and Custom methods unique to Tanssi.
+---
+
+# JSON-RPC API Methods
+
+## Custom JSON-RPC Methods {: #custom-json-rpc-methods }
```

Comment:
I'd swap the two sections so the standard ones come first

Reply:
great idea! done